src/basics/slowa.ts

```diff
@@ -0,0 +1,10 @@
+// // zdanie zrobic palindrom i anagram
+
+
+function palindrome(tablica: string []) { 
+    const odwrocone = slowo.split('').reverse().join('');
```

Comment:
Ogólnie nie powinno się używać w funkcji żadnych zmiennych, które są na zewnątrz niej, jeśli nie masz konkretnego powodu. Tutaj go nie masz, więc jeśli potrzebujesz czegoś wewnątrz tej funkcji, przekaż to jako argument.

Reply:
Oczywiście chodzi o zmienną `slowo`, która jest tutaj przez Ciebie wciągnięta spoza zakresu funkcji.